gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/secrets/conf/SSLUtils.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.apim.integration.tests.secrets.conf;
```

Comment:
should it be part of the sdk package to be reused by plugin tests later ?

Reply:
Yes it could be, it is rather generic way to create cert/key pairs. I can move it.